lib/linguist/languages.yml

```diff
@@ -4013,6 +4013,12 @@ Processing:
   tm_scope: source.processing
   ace_mode: text
   language_id: 294
+Proguard:
+  type: programming
```

Comment:
Should this really be counted as programming? I've always seen Proguard files as some sort of configuration files.

Reply:
Right. Changed to `data`.